test/utils/utils.js

```diff
@@ -56,7 +57,7 @@ function getJHipsterCli() {
     return cmd;
 }
 
-function testInTempDir(cb) {
+function testInTempDir(cb, keepInTestDir = false) {
```

Comment:
Wherever this function is used `keepInTestDir` is always `true`. So maybe remove that argument altogether?

Reply:
```suggestion
function testInTempDir(cb, keepInTestDir) {
```

Reply:
I meant to say it can be simplified by removing the argument itself; and this will enable removing condition on line 71 through 73.

Reply:
That will break the proposed change.

Reply:
You are right. I somehow assumed that that function is only used in `utils.js`. :smile:

Reply:
I think I understood now.
This can be done if every use of testInTempDir is covered.